tips/tests/tips_views_tests.py

```diff
@@ -82,26 +95,28 @@ def test_bad_form(self, generate_bad_form):
 class TestTipViews:
     def test_tips_board_function_200(self, generate_get_request, generate_tip):
```

Comment:
you need to fix this test, or @Yoavsc129 does that?

Reply:
the fix is merged